ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_IsType_User()
+        {
+            //Arrange
+            User _user = new User("Tommy");           
+            //Act
+            Article _article = new Article(_user);
+            //Assert            
+            Assert.IsType<User>(_article.Autor);
```

Comment:
do not use the 'IsType<>()` method if is not possible to fail the test here.
Check that `Author` property is changed and not null.
```suggestion
            Assert.NotNull(_article.Autor);
```